cluster-scope/overlays/prod/moc/infra/externalsecrets/thanos-object-storage.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: thanos-object-storage
```

Comment:
It seems the NESE on Infra is a longer journey than expected.. let's point it to AWS for now..

https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html-single/observability/index#enable-observability:~:text=For%20Amazon%20S3%20or%20S3%20compatible%2C%20your%20secret%20might%20resemble%20the%20following%20file%3A

https://github.com/operate-first/apps/pull/2286#pullrequestreview-1091209654

Reply:
@tumido, noted, I have updated [my follow on action comment](https://github.com/operate-first/apps/pull/2338#issuecomment-1231934112). That being said this doesn't change any of the manifests in this pr, simply my steps after this gets merged. Can I confirm that you mean Balrog s3 when you say "lets point it to AWS for now..."?

Reply:
I mean let's create a new AWS bucket + creds in the name of the operate first account and point it to there. I've just created one for you, please verify:

https://vault-ui-vault.apps.smaug.na.operate-first.cloud/ui/vault/secrets/k8s_secrets/show/moc/infra/open-cluster-management-observability/thanos-object-storage